pkg/skyenv/values.go

```diff
@@ -8,8 +8,9 @@ import (
 
 // Constants for skywire root directories.
 const (
-	DefaultSkywirePath = "."
-	PackageSkywirePath = "/opt/skywire"
+	DefaultSkywirePath        = "."
+	PackageSkywirePath        = "/opt/skywire"
+	DefaultServersListAddress = "http://servers-list.skycoin.com"
```

Comment:
I will speak to Murphy about the actual name of the domain. Add a TODO here for now please.

Reply:
Added